src/invitation.html

```diff
@@ -0,0 +1,114 @@
+---
```

Comment:
Can we call the file `visa-invitation.html` so future organizers will have an idea what this is about without having to open the file?